app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorEffectHandler.kt

```diff
@@ -28,8 +30,18 @@ class SyncIndicatorEffectHandler @AssistedInject constructor(
           .subtypeEffectHandler<SyncIndicatorEffect, SyncIndicatorEvent>()
           .addTransformer(FetchLastSyncedStatus::class.java, fetchLastSyncedState())
           .addTransformer(FetchDataForSyncIndicatorState::class.java, fetchDataForSyncIndicatorState())
+          .addTransformer(StartSyncedStateTimer::class.java, startTimer())
           .build()
 
+  private fun startTimer(): ObservableTransformer<StartSyncedStateTimer, SyncIndicatorEvent> {
+    return ObservableTransformer { effect ->
+      effect
+          .flatMap { Observable.interval(it.intervalAmount, it.timeUnit) }
+          .observeOn(schedulersProvider.ui())
```

Comment:
Do we need to switch the thread here anymore since we are just emitting an event?